android-template/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Tue Jan 30 13:14:22 CST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
```

Comment:
Do you know why we need the `all` distribution (with gradle source code) vs `bin`?

cc @jcesarmobile

Reply:
As you mentioned, `all` includes additional sources that makes it easier to work with Gradle scripts. On top of that, it's the default distribution that is used by new projects generated with Android Studio, as well as new projects initialized with the React Native CLI. That makes it much more likely to already exist in the local cache on a developer's machine, saving an additional download.